### PR TITLE
Speed up using the host composer cache

### PR DESCRIPTION
**Pull request by itsazzad**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ services:
         SYMFONY_VERSION: ${SYMFONY_VERSION:-}
         STABILITY: ${STABILITY:-stable}
     volumes:
+      - ~/.composer/cache:/root/.composer/cache
```
Comment (dunglas) [inline docker-compose.yaml]: Will it work on Windows?

Comment (itsazzad) [reply docker-compose.yaml]: Not tested

Comment (dunglas) [reply docker-compose.yaml]: I fear it will not.

**Comment by maxhelias** [comment]
This doesn't work on Windows, and there is currently no solution for compatibility between the different envs.

**Pull request closed without merging**